lazer/publisher_sdk/proto/publisher_update.proto

```diff
@@ -54,9 +54,18 @@ message PriceUpdate {
 
 // feed update containing data relating to funding rate
 message FundingRateUpdate {
+    enum FundingRateInterval {
+      INTERVAL_1H = 0;
+      INTERVAL_4H = 1;
+      INTERVAL_8H = 2;
+    }
```

Comment:
Doesn't seem like a good idea to restrict the possible set of values here. Why not use a Duration?

Reply:
yeah, maybe in the future people have half an hour too :D